content/https/https-3/contents.lr

```diff
@@ -0,0 +1,14 @@
+_model: question
+---
+title: What's the difference between Tor and DoH (DNS over HTTPS)?
+---
+seo_slug: difference-between-Tor-DoH
+---
+description:
+DNS over HTTPS (DoH) is a protocol for performing remote DNS resolution via the HTTPS protocol.

```

Comment:
Someone who doesn't know what DoH is is also unlikely to know what "remote DNS resolution" is.  I'm not sure i even know what "remote DNS resolution" is (at least i don't know what work the word "remote" is supposed to be doing here), and i work on these protocols.